CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,6 @@
-- [cosmos] [HARDENING] Add documentation about how to consume context data from Sinfonier (#145)
-- [cosmos] [HARDENING] Add Apiary documentation (#177)
-- [cosmos] [HARDENING] Add improvements to data_copier.sh (#180)
+- [cosmos][hardening] Add documentation about how to consume context data from Sinfonier (#145)
+- [cosmos][hardening] Add Apiary documentation (#177)
+- [cosmos-tidoop-api][feature] Add suport for MR job input parameters (#181)
+- [cosmos][hardening] Add documentation about how to consume context data from Sinfonier (#145)
+- [cosmos-tidoop-api][hardening]] Add Apiary documentation (#177)
+- [cosmos-admin][hardening] Add improvements to data_copier.sh (#180)
```

Comment:
I understand (by the PR tittle and the branch name) that this PR addresses #181. However, the diff also shows more additions... is this correct? Sometimes it is due to some merging problem.